### ISSUE TITLE
[A11y]Organization name and email fields are mandatory but not represented as mandatory fields.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8492]
Link: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1296190

​## Environment Details:
Application Name: NuGet Service
#URL: https://www.nuget.org
Operating System: Windows 10
Browser Version: Chrome Version 85.0.4183.121 (Official Build) (64-bit),
Chromium Edge Version 87.0.637.0 (Official build) dev (64-bit)

## Repro Steps:​
 Hit the URL " https://www.nuget.org" and login with appropriate credentials.
Tab Navigate to "Sign in" control and hit enter.
 Tab Navigate to "Sign in with Microsoft" control and hit enter.
Select “Manage Organizations” from the Profile Dropdown and hit enter.
Tab navigate to Add new link and hit enter.
Tab navigate to Organization name and email fields and observe these fields are mandatory but not provided as required field.
## Actual:​
Tab navigate to Organization name and email fields are mandatory but not represented as mandatory fields.
## Expected:​
Tab navigate to Organization name and email fields should be represented as mandatory fields with the required keyword.
​## User Impact:​
Keyboard only user will not not understand that field is mandatory if field is not marked with required keyword.

### Notes
We should update these fields to follow whatever pattern we use for other required field (asterisk, etc)
